package.json

```diff
@@ -115,58 +115,37 @@
     "packages/*/index.js",
     "packages/*/lib/**/*",
     "packages/*/src/**/*",
+    "packages/node_modules/**/*.js",
+    "packages/node_modules/**/*.ts",
```

Comment:
Why are we adding `.ts` files to the package?

Reply:
It's explained in a comment in `rspack.config.js`, but basically we expose types from some of our dependencies in our own type definition file, so we need to include their types in that scenario.